deployment/refinery-modules/refinery/manifests/selenium.pp

```diff
@@ -1,6 +1,6 @@
 class refinery::selenium {
   $geckodriver_version = 'v0.15.0'
-  $filename = "geckodriver-${geckodriver_version}-linux32.tar.gz"
+  $filename = "geckodriver-${geckodriver_version}-linux64.tar.gz"
```

Comment:
Because Geckodriver is not providing 32 bit builds in its newer releases anymore